react-app/src/components/Layout/index.tsx

```diff
@@ -162,8 +162,17 @@ const UserDropdownMenu = () => {
 export const Layout = () => {
   const isDesktop = useMediaQuery("(min-width: 768px)");
   const { data: user } = useGetUser();
+  const navigate = useNavigate();
   const customUserRoles = user?.user?.["custom:cms-roles"];
 
+  useEffect(() => {
+    const notLoggedOutPages =
+      window.location.pathname !== "/login" && window.location.pathname !== "/faq";
+    if (!user?.user && notLoggedOutPages) {
+      navigate("/login");
+    }
+  }, [user, navigate]);
```

Comment:
What're we doing here? I suspect this logic can be moved to a [`loader`](https://dev.to/shaancodes/a-brief-intro-about-loaders-in-react-router-54d).